SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -60,6 +62,29 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 		return NetDriver->StaticComponentView->HasAuthority(EntityId, Info->SchemaComponents[SCHEMA_ClientRPC]);
 	}
 
+	FORCEINLINE bool IsOwnedByWorker() const
```

Comment:
I find this name a little confusing for some reason. Maybe `IsOwnedByLocalWorker` or something? I'm unsure about that too :P